packages/klighd-core/src/search-bar/searchbar-actions.ts

```diff
@@ -0,0 +1,498 @@
+import { Action, CenterAction, SetModelAction, SModelElement, UpdateModelAction } from 'sprotty-protocol'

```

Comment:
Please add the EPL-2.0 license header to all of your new files like you can see in all other .ts(x) files in this repo.